tools/config/winlogbeat-modules-enabled.yml

```diff
@@ -225,7 +225,6 @@ fieldmappings:
     Accesses: winlog.event_data.Accesses
     AccessList: winlog.event_data.AccessList
     AttributeValue: winlog.event_data.AttributeValue
-    AttributeValue: winlog.event_data.AttributeValue
```

Comment:
Why did you remove these field mappings?

Reply:
Duplicate keys invalidate YAML

Reply:
Okay, thanks